### ISSUE TITLE
False positive "Invalid use paired tag without closing tag"

### ISSUE THREAD

**Issue by lulco** [mesour-intellij-plugins/latte-pro-issues#22]
**Describe the bug**
False positive "Invalid use paired tag without closing tag" + '[Latte] T_CLOSE_TAG' or '[Latte] T_CLOSE_TAG_EMPTY' expected, got 'null'

**Environment (please complete the following information):**
 - PhpStorm version: 2022.2
 - Plugin version: 1.0.2

**To Reproduce**
Steps to reproduce:
1. Use `null` as value for filter parameter

**Code**
```
{if $trailer->video}
    {$trailer->video|videoPlayer, null}
{/if}
```

**Comment by lulco** [comment]
Maybe it is related to https://github.com/mesour-intellij-plugins/latte-pro-issues/issues/20
If I change `,` to `:` it seems it works

**Comment by mesour** [comment]
@lulco yes, this is duplicity. But syntax error is fixed in 1.0.3
